cwms-data-api/src/main/java/cwms/cda/data/dao/ClobDao.java

```diff
@@ -269,11 +240,7 @@ public void getClob(String clobId, String officeId, ClobConsumer clobConsumer) {
                 try (ResultSet resultSet = preparedStatement.executeQuery()) {
                     if (resultSet.next()) {
                         java.sql.Clob clob = resultSet.getClob("VALUE");
-                        if (clob != null) {
-                            clobConsumer.accept(clob);
-                        } else {
-                            clobConsumer.accept(null);
-                        }
+                        clobConsumer.accept(clob);
```

Comment:
are we sure null here isn't valid?

Reply:
I think it is valid.  Its just that when clob == null the variable clob already has null in it so clobConsumer.accept(clob) is the right line in either case.   I didn't spot this myself, the IDE suggested it.  More of a - 'Huh - its right. ok.'

Reply:
oh right we're just passing null anyways.